docs/UI/CancelInteraction/assets/CancelInteractionSpecificAlert.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+
+participant Mobile
+participant Core
+participant HMI
+
+Mobile -> Core : CancelInteractionRequest\n(appID, ALERT, cancelID)
+
+Core -> HMI : CancelInteractionRequest\n(appID, ALERT, cancelID)
+
+HMI -> HMI : HMI Closes an active ALERT with\nthe corresponding cancel ID\nresponding with success if an\nALERT was closed
```

Comment:
Should probably have a separate diagram for the success case and failure (IGNORED?) case, rather than wrapping them both into this description.

Reply:
Broke out the two cases and made them real scenarios in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0